src/main/java/com/epam/reportportal/extension/azure/AzureExtension.java

```diff
@@ -231,19 +233,40 @@ private void removeListeners() {
 
 	@Override
 	// Never called method. Connection is tested via the command.
-	public boolean testConnection(Integration system) {
+	public boolean testConnection(Integration integration) {
 		return false;
 	}
 
 	@Override
-	// TODO: Implement in the future
-	public Optional<Ticket> getTicket(String id, Integration system) {
-		return Optional.empty();
+	public Optional<Ticket> getTicket(String id, Integration integration) {
+		Map<String, Object> params = integration.getParams().getParams();
+		String organizationUrl = params.get("url").toString();
+		String projectName = params.get("project").toString();
+		String personalAccessToken = params.get("oauthAccessKey").toString();
+
+		ApiClient defaultClient = getConfiguredApiClient(personalAccessToken);
+		String organizationName = organizationUrl.replace(defaultClient.getBasePath(), "");
```

Comment:
can this be a function? are all clients have basePath?

Reply:
Introduced the `extractOrganizationNameFromUrl()` method for this